deletion/deleter.go

```diff
@@ -0,0 +1,166 @@
+package deletion
+
+import (
+	"errors"
+	"github.com/hashicorp/go-multierror"
+	"github.com/op/go-logging"
+	errors2 "github.com/pkg/errors"
+	"io"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+var log = logging.MustGetLogger("deletion")
```

Comment:
Move this to the var group for consistency.

Reply:
done.